Doc/source/pybats.rst

```diff
@@ -1,7 +1,347 @@
-
 ########################################
 PyBats - SWMF & BATS-R-US Analysis Tools
 ########################################
 
-.. currentmodule:: spacepy.pybats
-.. automodule:: spacepy.pybats
+.. contents:: Table of Contents
+    :depth: 2
+    :local:
+
+PyBats!  An open source Python-based interface for reading, manipulating,
+and visualizing BATS-R-US and SWMF output.
+For more information on the SWMF, please visit the
+`Center for Space Environment Modeling <http://csem.engin.umich.edu>`_.
+
+See also the `full API documentation <spacepy.pybats>`.
+
+PyBats Module
+^^^^^^^^^^^^^
+
+pybats
+------
```

Comment:
Follow the [Sphinx conventions](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-sections) for header ordering. No need to rework all the existing stuff in this order, but where we're adding new headers, try to do that.

Each of these submodule-specific sections should link to that submodule, and the submodule should contain a `:ref:` link back to these sections.

Reply:
Yeah, when I was going through originally I was looking at the sphinx conventions, it didn't seem to work the way it should, this weirdness with the ^^^^^ and the ------- was due to sphinx not rendering the sections correctly any other way.

Reply:
It's going to depend a bit on what other .rst files are doing. Stick to the sphinx ordering even if the ToC looks nested wrong) and we'll double back later to make them consistent across all the docs.

Reply:
Okay, makes sense to stick to conventions, I can do that.